<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categorybudgeting</class>
 <widget class="QDialog" name="categorybudgeting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>221</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setlimitbtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category Based Budgeting</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>233</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select Category</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="categorycomboBox">
      <item>
       <property name="text">
        <string>Food</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Entertainment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Travelling</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Clothing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Education Equipment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>University Fees</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>201</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Set / Update Limit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View Your Limits</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
